cmd/orpheus/controller_discord.go

```diff
@@ -24,6 +24,16 @@ var commands = []*discordgo.ApplicationCommand{
 			},
 		},
 	},
+	{Name: "addshuf", Description: "Adds a playlist to the queue in a shuffled order",
```

Comment:
Instead of there being `add` and `addshuf`, I think it may make more intuitive sense for there to be `add` and `addlist`, where shuffle is an optional parameter to addlist?

Reply:
We should also probably guarantee that `add` only adds a single song, to make the intention of the user extra clear. You may be on a song that just happens to be on a playlist and add like 100 songs unintentionally.